lib/recipe/bloc/recipe_bloc.dart

```diff
@@ -210,6 +211,17 @@ class RecipeBloc extends Bloc<RecipeEvent, RecipeState> {
     emit(state.copyWith(recipes: recipes));
   }
 
+  bool _checkProbesChecked(List<Probe> probes) {
+    bool allProbesChecked = true;
+    for (Probe probe in probes) {
+      if (probe.checked == false) {
+        allProbesChecked = false;
+        break;
+      }
+    }
+    return allProbesChecked;
+  }
+
```

Comment:
Will this be used to check a subset of the probes at some point? If not, it might be nicer to rename it to `_areProbesChecked()` or `_areAllProbesChecked()` and perhaps also move it to a method of the `RecipeState` so that it could be called without any arguments like `bool allProbesChecked = state.areProbesChecked()`

Reply:
Obviously this is very minor — I just thought checkProbesChecked could be made a little better sounding by making the `check` part some other verb like `are` or something similar, but obviously this is very very minor suggestion 😄.

Reply:
Sounds good :)

Reply:
Moving it to a state method doesn't quite work given `RecipeState` currently holds the list of recipes. An alternative is to have this as a method of the recipe model itself - which then makes it a choice of whether we want this logic to reside in the bloc or in the model as a method of the class. I think either works fine but am going to leave it in the bloc for now.

Reply:
Ahh so you mean that since `RecipeState` holds a **_list_** of recipes, we would need to specify which recipe we want to check for the probes, right? so instead of `state.areProbesChecked()` it would need to be something like `state.areProbesOfRecipeChecked(Recipe)` or something like that?

Yeah anyway it looks good now! 👍